src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -438,8 +454,11 @@ export class MeetingManager implements AudioVideoObserver {
     }
   }
 
-  selectAudioInputDevice = async (deviceId: string): Promise<void> => {
-    const receivedDevice = audioInputSelectionToDevice(deviceId);
+  selectAudioInputDevice = async (device: Device | AudioTransformDevice, is440: boolean = false): Promise<void> => {
```

Comment:
Suggest to remove the extra parameter and check if we could handle this internally in the method. Any reason for this param addition?

Reply:
Based on our discussion, can not come up with a good idea to get rid of this parameter. Need help with team.

Reply:
```typescript
foo.selectAudioInputDevice(DefaultDeviceController.synthesizeAudioDevice(440));
```

If you can't do that, it is a strong indicator that the API needs work: `Device` includes `MediaStream`, and so you just need a way for the builder to create one and pass it in here.

Reply:
Changed this parameter `is440` to `innerDeviceId` to indicate the inner device (string) before synthesize.

Reply:
Based on our discussion, removed this parameter. Add some description on this `440 Hz` in the documentation of `AudioInputVFControl`